applications/FluidDynamicsApplication/python_scripts/turbulence_model_solver.py

```diff
@@ -17,53 +17,53 @@ def CreateTurbulenceModel(model, settings, is_distributed = False):
+        msg = "Calling the base TurbulenceModelSolver class AddVariables method."
         msg += " Please override it in the derrived class."
         raise Exception(msg)
 
     def AddDofs(self):
-        msg = "Calling the base TurbulenceModelConfiguration class AddDofs method."
+        msg = "Calling the base TurbulenceModelSolver class AddDofs method."
         msg += " Please override it in the derrived class."
         raise Exception(msg)
 
     def PrepareModelPart(self):
-        msg = "Calling the base TurbulenceModelConfiguration class PrepareModelPart method."
+        msg = "Calling the base TurbulenceModelSolver class PrepareModelPart method."
         msg += " Please override it in the derrived class."
         raise Exception(msg)
 
     def GetTurbulenceSolvingProcess(self):
-        msg = "Calling the base TurbulenceModelConfiguration class GetTurbulenceSolvingProcess method."
+        msg = "Calling the base TurbulenceModelSolver class GetTurbulenceSolvingProcess method."
         msg += " Please override it in the derrived class to return a KratosMultiphysics.Process."
         raise Exception(msg)
 
     def Initialize(self):
-        msg = "Calling the base TurbulenceModelConfiguration class Initialize method."
+        msg = "Calling the base TurbulenceModelSolver class Initialize method."
         msg += " Please override it in the derrived class."
         raise Exception(msg)
 
     def Check(self):
-        msg = "Calling the base TurbulenceModelConfiguration class Check method."
+        msg = "Calling the base TurbulenceModelSolver class Check method."
         msg += " Please override it in the derrived class."
         raise Exception(msg)
 
     def InitializeSolutionStep(self):
-        msg = "Calling the base TurbulenceModelConfiguration class InitializeSolutionStep method."
+        msg = "Calling the base TurbulenceModelSolver class InitializeSolutionStep method."
         msg += " Please override it in the derrived class."
         raise Exception(msg)
 
     def FinalizeSolutionStep(self):
-        msg = "Calling the base TurbulenceModelConfiguration class FinalizeSolutionStep method."
+        msg = "Calling the base TurbulenceModelSolver class FinalizeSolutionStep method."
         msg += " Please override it in the derrived class."
         raise Exception(msg)
 
     def GetFluidVelocityPressureConditionName(self):
-        msg = "Calling the base TurbulenceModelConfiguration class GetFluidVelocityPressureConditionName method."
+        msg = "Calling the base TurbulenceModelSolver class GetFluidVelocityPressureConditionName method."
         msg += " Please override it in the derrived class to return a condition name."
         raise Exception(msg)
 
     def SetCommunicator(self, current_communicator):
-        msg = "Calling the base TurbulenceModelConfiguration class SetCommunicator method."
+        msg = "Calling the base TurbulenceModelSolver class SetCommunicator method."
```

Comment:
what communicator is this?
Kratos Communicator?
Kratos Data-Communicator?
Epetra Communicator?
MPI_COMM?

Reply:
done... its the epetra_communicator